### DISCUSSION TITLE
Is it possible to cancel a download?

### DISCUSSION THREAD

**Discussion by lumpiluk** [UnicornsOnLSD/finamp#909] [General]
I tapped on the download button on a very large playlist to see if there was going to be a submenu (first-time user here). Now it seems I have to wait for a multi-GB download to finish before I will be able to free up this space on my device again. Is there any way to abort the download of a playlist?

**Comment by Chaphasilor** [comment]
You should be able to hit the "delete" button while it's still downloading to abort it. It could be that any download that has been submitted to the operating system already will not be canceled, but on Android at least those are only a couple of downloads.
